contracts/strategy/rysk/RyskStrategy.sol

```diff
@@ -12,6 +12,8 @@ import {BaseStrategy} from "../BaseStrategy.sol";
 import {IRyskLiquidityPool} from "../../interfaces/rysk/IRyskLiquidityPool.sol";
 import {IVault} from "../../vault/IVault.sol";
 
+import "hardhat/console.sol";
+
```

Comment:
```suggestion
```

Reply:
This is a draft, so I'm still using that. Please review the functionality.